lib/puppet-lint/bin.rb

```diff
@@ -74,6 +74,9 @@ def run
       puts "puppet-lint: no file specified or specified file does not exist"
       puts "puppet-lint: try 'puppet-lint --help' for more information"
       return 1
+    rescue ArgumentError
```

Comment:
What's raising the error? Are we depending on an implementation detail? Is this going to mask other errors?

ArgumentError seems overly broad.